samcli/lib/telemetry/event.py

```diff
@@ -134,7 +138,9 @@ class EventTracker:
     MAX_EVENTS: int = 50  # Maximum number of events to store before sending
 
     @staticmethod
-    def track_event(event_name: str, event_value: str):
+    def track_event(
+        event_name: str, event_value: str, session_id: Optional[str] = None, exception_name: Optional[str] = None
```

Comment:
why do we need to send the `session_id` is there any case that this parameter will be different than the `session_id` retrieved from the command context ?